x-team/xp-ui
# Clean up applicant-facing screens

**Release Type:** *Breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3484

## Description

Clean up applicant-facing screens

## Checklist


## Related PRs

branch | PR
------ | ------
`XP-3484-clean-up-applicant-facing-screens-code-from-xp-and-xp-ui` | [XP](https://github.com/x-team/xp/pull/2051)

## Steps to Test or Reproduce

Please refer to https://github.com/x-team/xp/pull/2051. As long nothing got broken in XP-UI, in this PR there's nothing really to see beyond the `MilestonesScreen` story that doesn't exist anymore in the Storybook.

## Impacted Areas in Application

Just `MilestonesScreen` got removed and related stories file.
